squad-server/plugins/db-log.js

```diff
@@ -146,6 +147,41 @@ export default class DBLog extends BasePlugin {
       }
     );
 
+    this.createModel(
+      'PlaySession',
+      {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+
+        steamID: {
+          type: DataTypes.STRING
+        },
+
+        lastName: {
+          type: DataTypes.STRING
+        },
```

Comment:
I'd suggest just using name - at some point I want to do the same for the SteamUser table.